client/landing/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -0,0 +1,48 @@
+// Menu links
+.sidebar__menu-link {
+	padding-top: 14px;
+	padding-bottom: 14px;
+	font-weight: 600;
+
+	.selected &,
+	.selected &:hover {
+		background-color: rgba( var( --color-sidebar-menu-selected-background-rgb ), 0.12 );
+	}
+}
+
+// Special case for the "Manage site" menu item icon
+.sidebar__menu-item-back .material-icon {
+	transform: scaleX( -1 );
+}
+
+// Expandables
+.sidebar__menu.is-togglable {
+	.sidebar__heading {
+		padding-top: 15px;
+		padding-bottom: 15px;
+	}
+
+	&.is-toggle-open {
+		.sidebar__heading {
+			background-color: var( --studio-white );
+		}
+	}
+}
+
+.sidebar__expandable-content {
+	.sidebar__menu-link {
+		margin: 2px 8px;
```

Comment:
Not sure why, but it looks much more natural to me when we remove this margin.

Reply:
Yeah, it may seem a bit odd, but that's how the design looks. Let's maybe double check with @Automattic/jetpack-design if this margin is intended?

Reply:
Agreed, I think it looks much better without it.